shared/src/arguments.rs

```diff
@@ -73,6 +73,14 @@ pub struct Arguments {
     #[structopt(long, env, default_value = "4")]
     pub pool_cache_maximum_recent_block_age: u64,
 
+    /// How often to retry requests in the pool cache.
+    #[structopt(long, env, default_value = "5")]
+    pub pool_cache_maximum_retries: u32,
```

Comment:
Note that without retrying we wouldn't get a response at all so waiting up to ~5 seconds is still better than nothing.

Reply:
~~We might want to include "seconds" in the name here, I interpreted this as number of retries. unless that's what it is... but the doc string seems to conflict with the name.~~

Given the next config value, I am beginning to suspect it is the doc string that is incorrect here.